lib/livebook_web/live/settings_live/autosave_path_component.ex

```diff
@@ -0,0 +1,36 @@
+defmodule LivebookWeb.SettingsLive.AutosavePathComponent do
+  use LivebookWeb, :live_component
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <div class="flex items-center justify-between border border-gray-200 rounded-lg p-4">
+        <div class="h-full h-52">
+          <.live_component module={LivebookWeb.FileSelectComponent}
+              id="autosave-path-component"
+              file={@autosave_file}
+              extnames={[]}
+              running_files={[]}
+              submit_event={:init}
+              file_system_select_disabled={true} />
+        </div>
+      </div>
+      <div class="flex">
+        <button
+          class="button-base button-blue mt-4 mr-4"
+          phx-click="set_autosave_path"
+        >
+          Save
+        </button>
```

Comment:
We can render those buttons on the right side of the input box, by wrapping them with in the component [like this](https://github.com/livebook-dev/livebook/blob/d840a3b99db845e70d398ac7dd7a8e62bca4f45b/lib/livebook_web/live/session_live/persistence_live.ex#L107-L126).